apps/web/pages/[user].tsx

```diff
@@ -139,12 +191,60 @@ export default function User(props: inferSSRProps<typeof getServerSideProps>) {
 export const getServerSideProps = async (context: GetServerSidePropsContext) => {
   const ssr = await ssrInit(context);
   const crypto = require("crypto");
+  const getEventTypesWithHiddenFromDB = async (userId: Number) => {
```

Comment:
We can move this arrow function outside getServerSideProps so it gets cleaner, also remember to pass all required params as it looks to me its reading `take: users[0].plan === UserPlan.FREE ? 1 : undefined` from runtime.

Reply:
Good catch. Will pass the `plan` as param too. Thanks!!